### PR TITLE
Add option to include sample content

### PR DESCRIPTION
**Pull request by danlamanna**

### CODE REVIEW

File: hooks/post_gen_project.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+import os
+import shutil
+
+PROJECT_DIRECTORY = os.path.realpath(os.path.curdir)
+
+SAMPLE_MODEL_FILES = [
+    os.path.join(
```
Comment (brianhelba) [inline hooks/post_gen_project.py:8]: Could this use `pathlib.Path` instead? It'd be more elegant to manipulate paths.

**Comment by brianhelba** [comment]
@danlamanna Could you rebase this? I'd like to get it merged.

**Comment by danlamanna** [comment]
@brianhelba ready

**Comment by brianhelba** [comment]
Replaced by #111

**Pull request closed without merging**